tools/simd.cmake

```diff
@@ -0,0 +1,39 @@
+include(CheckCXXSourceRuns)
```

Comment:
I suggest renaming the file to something like `msvc_isa.cmake` since it only affects the MSVC compiler.